base/compiler/types.jl

```diff
@@ -122,14 +122,15 @@ mutable struct InferenceResult
     overridden_by_const::BitVector
     result                   # ::Type, or InferenceState if WIP
     src                      # ::Union{CodeInfo, OptimizationState} if inferred copy is available, nothing otherwise
+    edges::Vector{Any}       # backedges for this result
```

Comment:
This seems a bit out of place here, since the others are either the keys describing this, or the inferred types. This is neither (it is internal state of the optimizer)